packages/react/mixins/mixin.browser.js

```diff
@@ -24,9 +25,12 @@ class ReactMixin extends Mixin {
     this.element = element;
   }
   enhanceElement(element) {
+    const basename = ensureLeadingSlash(
+      trimTrailingSlash(this.config.basePath)
+    );
     const props = {
       ...this.options.router,
-      basename: this.config.basePath,
+      basename: basename === '/' ? '' : basename,
```

Comment:
Would switching the `pathifist` helper application order not allow us to remove this extra step..?

```
> trimTrailingSlash(ensureLeadingSlash('/foo/'))
'/foo'
> trimTrailingSlash(ensureLeadingSlash('/foo'))
'/foo'
> trimTrailingSlash(ensureLeadingSlash('foo/'))
'/foo'
> trimTrailingSlash(ensureLeadingSlash('/'))
''
```